src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -145,6 +147,22 @@ const Body = BaseFormWithPolling.extend(
       });
     },
 
+    showCustomFormErrorCallout(error) {
+      const errorSummary = error.responseJSON.errorSummary;
+      let options = {
+        type: 'error',
+        className: 'okta-verify-uv-callout-content',
+        subtitle: errorSummary,
+      };
+
+      if (errorSummary.includes(INVALIDATED_ACCOUNT_DEVICE_ERR_MSG)) {
+        options.title = loc('user.fail.verifyIdentity', 'login');
+      }
```

Comment:
It is very unlikely this will always be an English error message. Can you please rely on an `errorCode` instead?

Reply:
+1

Reply:
Updated